### ISSUE TITLE
References to non-existent package ...mpa.model

### ISSUE THREAD

**Issue by Alfiva** [universAAL/middleware#491]
I tried building the latest 3.4.2-S but found out some .osgi bundles are trying to export or import the package org.universAAL.middleware.interfaces.mpa.model, which no longer exists in mw.interfaces.core. The failing bundles are mw.interfaces.osgi and mw.connectors.deploy.karaf.osgi. Can you check if it is OK with just removing the mention to that package in their POM? I did that and succeeded but not sure if anything else should be modified.

BTW I also had to add org.universAAL.support/itests dependency to mw.managers.tenant.osgi POM to make it build OK.

**Comment by cstockloew** [comment]
I removed that package almost a year ago. It was not used at all and I thought I removed all references to it..

There is another package org.universAAL.middleware.managers.deploy.uapp.model in mw.schemas with almost the same content (the same classes). It seemed to me that the one in mw.schemas was more up-to-date than the one in mw.interfaces and that's why I removed the 'old' one.

However, I'm not sure why the build is failing in your case; it works on my PC and it works on CI.

OK for removing..
